packages/faq-extension/faq.md

```diff
@@ -7,20 +8,31 @@
 
 ## Development
 
-* [How do I report a bug or provide feedback?](#How-do-I-report-a-bug-or-provide-feedback?)
-* [How can I contribute?](#How-can-I-contribute?)
-* [How can I extend or customize JupyterLab?](#How-can-I-extend-or-customize-JupyterLab?)
+* [How can you report a bug or provide feedback?](#How-can-you-report-a-bug-or-provide-feedback?)
+* [How can you contribute?](#How-can-you-contribute?)
+* [How can you extend or customize JupyterLab?](#How-can-you-extend-or-customize-JupyterLab?)
 
 ## General
 
+Welcome to the Frequently Asked Questions (FAQ) for the JupyterLab Beta Release.
+
 ### What is JupyterLab?
 
-JupyterLab is a new user interface for the Jupyter Notebook, allowing
-users to arrange multiple Jupyter notebooks, text editors, terminals,
-output areas, etc. on a single page with multiple panels and tabs in
-one integrated application. The codebase and user-interface of JupyterLab
-is based on a flexible extension system that makes it easy to extend with new
-functionality.
+JupyterLab is a new user interface for Project Jupyter, allowing users to
+arrange multiple Jupyter notebooks, file editors, terminals, output areas,
+etc. on a single page with multiple panels and tabs in one integrated
+application. While JupyterLab looks like an IDE (Integrated Development
+Environment) it remains focused on the core Jupyter experience of
+interactive computing with data.
+
+In addition to the Jupyter Notebook, JupyterLab offers multiple models of
+interactive computing, including a scratchpad-like Code Console and the
+ability to attach kernels to any text file. JupyterLab has
+built-in support for many common file and outputs formats (CVS, PDF, Vega,
```

Comment:
`file and output formats` (singularlize 'outputs')

Reply:
fixed